tools/config-gen/params.yaml

```diff
@@ -37,3 +38,24 @@
   config-path: "debug.log-mutex"
   type: "bool"
   usage: "Print debug messages when a mutex is held too long."
+
+- flag-name: "log-file"
+  config-path: "logging.file-path"
+  type: "resolvedPath"
+  usage: >-
```

Comment:
Why is >- needed?

Reply:
">" is one of the ways to write multiline strings in YAML. The other is "|". While "|" preserves newline, ">" doesn't. However, it adds a newline character at the end of the string. 

So,
a: >
  Hello
  World

is semantically equivalent to
a := "Hello World\n"

We don't need to have an additional newline character in our usage doc. So, I added the "-" sign which does nothing except removing the trailing newline.